apps/fees-pay/ccpay-cpo-update-service/perftest.yaml

```diff
@@ -5,6 +5,14 @@ metadata:
 spec:
   values:
     java:
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: '1024Mi'
```

Comment:
Same story here. If memory is expected to stay constant, scaling based on memory can be disabled, or avg. utilization increased.